.gitignore

```diff
@@ -68,3 +68,4 @@ build/
 ## OS Specific
 .DS_Store
 Thumbs.db
+core/src/com/group66/game/cannon/Cannon.java
```

Comment:
No we don't
This is was done by mistake when solving a previous conflict error
Fixed now